content/01.proposal.md

```diff
@@ -0,0 +1,54 @@
+Over 30 authors contributed to the Deep Review [@doi:10.1101/142760], using [issues](https://github.com/greenelab/deep-review/issues?utf8=%E2%9C%93&q=is%3Aissue) to discuss source material and [pull requests](https://github.com/greenelab/deep-review/pulls?utf8=%E2%9C%93&q=is%3Apr) to propose manuscript changes.
+The community appreciated the breadth of perspectives on a quickly-developing topic and the Deep Review was the most-viewed _bioRxiv_ preprint from April–June 2017 [@doi:10.1038/d41586-017-08493-x].
+We've also used the Manubot to write our [Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/) and to reproduce the [Bitcoin Whitepaper](https://dhimmel.github.io/bitcoin-whitepaper/) [@tag:steemit-post].
+
+We are seeking support to fund Manubot development in 2018.
+Some **specific aims** that we're excited about are:
+
+1. JATS is the predominant archival format for scholarly manuscripts that is optimized for machine-readability and interoperability.
+We want the Manubot to **export to JATS** and to incorporate an existing **JATS viewer**, such as [Lens](https://lens.elifesciences.org/about/), into Manubot webpages.
+
+2. Currently all changes to a manuscript's source are tracked using git.
+However, Manubot does not automatically generate **diffs for rendered outputs**.
+By creating rendered diffs we can show how a manuscript changed from the reader's perspective and make creating "tracked changes" documents for journal resubmission easy.
+
+3. **End-to-end reproducibility** is when the provenance of every manuscript result can be traced to its source.
+Manubot allows users to pass template variables, so results can be embedded directly from the analysis that created them.
+We'd like to experiment more with the templating system and make it as user-friendly as possible.
+
+4. We have several ideas for making **Manubot's citation metadata** system even more powerful and versatile.
+We're confident that citing (only) by standard identifiers is the way of the future and Manubot can serve as a reference for other applications deciding to implement citation-by-ID.
+
+5. Currently Manubot requires considerable technical expertise [to setup](https://github.com/greenelab/manubot-rootstock/blob/f165f609f33b11fdf71a0db6435d4dd159f23973/SETUP.md) and some technical expertise [to use](https://github.com/greenelab/manubot-rootstock/blob/f165f609f33b11fdf71a0db6435d4dd159f23973/USAGE.md).
+Therefore, the target userbase is researchers or publishers who are already using tools like git and continuous integration for other tasks.
+Another continuing aim is therefore to support **outreach and user adoption**, as well as identifying which potential users will benefit most from the platform.
+
+Our **long-term vision** is an improved system of scholarly communication, brought about by tools such as Manubot.
```

Comment:
Suggest ordering: 
Our **long-term vision** is an improved ecosystem of scholarly communication.
Imagine you could click an author to highlight only the text they wrote.
Or highlight all results and figures that descend from source code contributions by the selected author.
Or highlight all results that depend on a selected upstream experiment or database.
Or if a new version of software used in the manuscript or an updated dataset is released, rebuild the manuscript with the updated version and see the resulting changes...
regardless of whether you're the original authors or a curious third-party!
Manubot forms the foundation for the traceability of all aspects of a scholarly manuscript, which is the key principle that underlies this potential future.

Reply:
Done in https://github.com/greenelab/manufund-2018/pull/2/commits/74605fed96e844030034db19f43c0cb3e69c52ca